### PR TITLE
Bug 1174444 - [Settings] Convert screen lock switches to use gaia-switch

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1174444

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d3c0e992cbf1c7d1a89276b539105b72c94beb7d

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=30a5e3a45d7e03c5253b0c5fd01e7de645262ead

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=138143296a949c67ac5ea35aca5b20b4dd6706f4

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=760f64fc068235aa66f64d5a7c10f2a8f6d7792a

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=408a52046a678515f9ab265e6c2d0dd33f30f46e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f19de0027428c15749c815aa995effe2b25d20ca

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/screen_lock.py [hunk 1]
```diff
@@ -10,8 +10,8 @@
 class ScreenLock(Base):
 
     _screen_lock_section_locator = (By.ID, 'screenLock')
-    _lockscreen_checkbox_locator = (By.XPATH, '//li/label[span[@data-l10n-id="lockScreen"]]')
-    _passcode_checkbox_locator = (By.XPATH, '//li/label[span[@data-l10n-id="passcode-lock"]]')
+    _lockscreen_checkbox_locator = (By.XPATH, '//li/gaia-switch/label[label[@data-l10n-id="lockScreen"]]')
```
Comment (crh0716) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/screen_lock.py]: should be '//li/gaia-switch/label[@data-l10n-id="lockScreen"]'?

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b2d2e3407f5f3a58825429964c12d7c3bb1a7608

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c1c13a2d969e9317f30e5ecb1b786fcd2f437a90

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7c0796047e89ec7d7f434acbf2f16b7d9ae28c07

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9d5cafcd66cf40061593fe00eb1be0d303708668

**Pull request merged**